hudi-sync/hudi-hive-sync/src/test/java/org/apache/hudi/hive/TestHiveSyncTool.java

```diff
@@ -88,8 +88,8 @@
 public class TestHiveSyncTool {
 
   private static final List<Object> SYNC_MODES = Arrays.asList(
-      "hiveql",
-      "hms",
+      /*"hiveql",
+      "hms",*/
```

Comment:
Any reason for commenting these?

Reply:
Good catch. This was just to reduce test time locally. I have removed the comment.